src/async_graphql_hyper.rs

```diff
@@ -16,6 +16,14 @@ impl GraphQLBatchRequest {
   {
     GraphQLResponse(executor.execute_batch(self.0).await)
   }
+
+  #[must_use]
```

Comment:
```suggestion
  #[must_use]
```
Add a comment to explain why `must_use` is required.